src/database/migrations/20200303093352-create-comments.js

```diff
@@ -0,0 +1,46 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Comments', {
+    id: {
+      allowNull: false,
+      primaryKey: true,
+      type: Sequelize.STRING
+    },
+    requestId: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
+        model: 'travelRequests',
+        key: 'id',
+        as: 'id',
+      }
+    },
+    userID: {
+      type: Sequelize.STRING,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+      references: {
+        model: 'Users',
+        key: 'id',
+        as: 'id',
+      },
+    },
+    comment: {
+      type: Sequelize.TEXT,
+      allowNull: false,
+    },
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    }
+  }),
+  // eslint-disable-next-line no-unused-vars
```

Comment:
@blaise82 Add the rule to ```.eslintIgnore`` or try to fix it like so

Reply:
I can't ignore that rule it helps sometimes
but let me just add specific files to eslintIgnore

Reply:
@blaise82 Sure thing